### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by thanhlmm** [subsquid/squid-cli#74]
When I run

```
npm i -g @subsquid/cli@latest
```

It returns error
```
npm ERR! code 1
npm ERR! path /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cli
npm ERR! command failed
npm ERR! command sh -c -- node ./bin/run.js autocomplete --refresh-cache
npm ERR! ›   ModuleLoadError: [MODULE_NOT_FOUND] require failed to load /Users/thanhle/
npm ERR!  ›   .nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cli/lib/commands: 
npm ERR!  ›   Cannot find module '/Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_mo
npm ERR!  ›   dules/@subsquid/cli/lib/commands'
npm ERR!  ›   Require stack:
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/module-loader.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/help/index.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/flags.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/cli-ux/styled/table.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/cli-ux/styled/index.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/cli-ux/index.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/node_modules/@oclif/core/lib/index.js
npm ERR!  ›   - /Users/thanhle/.nvm/versions/node/v16.20.2/lib/node_modules/@subsquid/cl
npm ERR!  ›   i/bin/run.js
npm ERR!  ›   Code: MODULE_NOT_FOUND
```

**Comment by belopash** [comment]
What is your setup? WSL?

**Comment by mlenser** [comment]
I have the same issue locally on windows and all of my CI experience the same issue.

**Comment by thanhlmm** [comment]
MacOS. The root cause of `@oclif/core` new version

**Comment by mlenser** [comment]
> The root cause of `@oclif/core` new version

If this is saying that `@oclif/core` is the root cause: I'm not sure. I resolved it to `3.20.0` (last week) and the issue still persists.

**Comment by thanhlmm** [comment]
> > The root cause of `@oclif/core` new version
> 
> If this is saying that `@oclif/core` is the root cause: I'm not sure. I resolved it to `3.20.0` (last week) and the issue still persists.

3.19.1 will work bro

**Comment by belopash** [comment]
Should be resolved in `subsquid/cli@2.8.7`. We locked oclif version for now.

**Comment by mlenser** [comment]
I can confirm the issue is resolved via `subsquid/cli@2.8.7`. Thank you for the quick turnaround!
